content/doc/administration/requirements/upgrade-java-guidelines.adoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: documentation
+title: Upgrading Jenkins Java version from 8 to 11
+---
+
+In order to upgrade the JVM used to run Jenkins, more specifically from Java 8 to Java 11, there are a few details and steps to take care.
+
+== Backup
+
+As for any upgrade, we recommend you to backup the `JENKINS_HOME`.
+It is also recommended to test the upgrade with the backup before performing it on the production instance.
+
+== Plugins upgrade
+
+First, we need to understand that some plugins were not directly compatible with Java 11.
+So, it is important you upgrade all the plugins to the latest version available at the time you are performing the upgrade.
```

Comment:
How about "Some plugins were not previously compatible with Java 11.  Therefore it is important that you upgrade all of your plugins..."  or maybe even "Many plugins were not previously compatible with Java 11".

Reply:
```suggestion
Therefore it is important that you upgrade all of your plugins.
```